frontend/src/themes/theme.js

```diff
@@ -19,7 +19,7 @@ const coreTheme = createMuiTheme({
   typography: {
     fontFamily: "Open Sans",
     h1: {
-      fontSize: "3rem",
+      fontSize: "calc(1.6rem + 6 * ((100vw - 320px) / 680))",
```

Comment:
Please revert this change as it might have unintended side-effects on other parts of the page (i.e. h1s being displayed differently than expected)